api/models/education.md

```diff
@@ -0,0 +1,86 @@
+# Brainstorm for API End Points of Education Model
```

Comment:
Looks great!

Reply:
These endpoints make sense for us. Survey will be called to populate our questions, and report will be called after questions are submitted.